### PR TITLE
Build nss with OS_TARGET=WINNT

### PR DESCRIPTION
**Pull request by Pilot-Pirx**

### CODE REVIEW

**Review by ardovm — CHANGES_REQUESTED** [review]
Did you try signing and verifying signatures with this build?
That should be a good check that everything works.

File: main/libxmlsec/xmlsec1-configure.patch [hunk 1]
```diff
@@ -759,7 +759,7 @@
  XMLSEC_OPENSSL_ALIBS    = libeay32.lib wsock32.lib kernel32.lib user32.lib gdi32.lib
  
 -XMLSEC_NSS_SOLIBS   	= smime3.lib ssl3.lib nss3.lib libnspr4.lib libplds4.lib libplc4.lib kernel32.lib user32.lib gdi32.lib
-+XMLSEC_NSS_SOLIBS   	= smime3.lib nss3.lib nspr4.lib kernel32.lib user32.lib gdi32.lib
++XMLSEC_NSS_SOLIBS   	= smime3.lib nss3.lib licnspr4.lib libplds4.lib libplc4.lib kernel32.lib user32.lib gdi32.lib
```
Comment (ardovm) [inline main/libxmlsec/xmlsec1-configure.patch]: This li*c*nspr4.lib looks like a typo...?

Comment (Pilot-Pirx) [reply main/libxmlsec/xmlsec1-configure.patch]: That's definitely a typo, but it did build anyway... ;-)

Comment (Pilot-Pirx) [reply main/libxmlsec/xmlsec1-configure.patch]: Fixed!

**Comment by Pilot-Pirx** [comment]
Please find a Windows build here:
https://www.dropbox.com/scl/fi/8kx1kwcet1ldnb5ckpiax/Apache_OpenOffice_4.5.0_Win_x86_install_en-US_nss.exe?rlkey=o4qh04sd805weh62h8iycdcsd&dl=0

**Comment by Pilot-Pirx** [comment]
> Did you try signing and verifying signatures with this build? That should be a good check that everything works.

I could successfully sign a document with my recent build.
[Signed document.odt](https://github.com/apache/openoffice/files/14185138/Signed.document.odt)

An older (signed) document can be opened without warning and the signature can be validated.

**Review by ardovm — APPROVED** [review]

**Comment by ardovm** [comment]
This PR looks good to me. Thank you!

**Pull request merged**